ui/app/models/clients/activity.js

```diff
@@ -1,10 +1,12 @@
 import Model, { attr } from '@ember-data/model';
 export default class Activity extends Model {
-  @attr('string') responseTimestamp;
+  @attr('array') byMonthTotalClients;
+  @attr('array') byMonthNewClients;
   @attr('array') byNamespace;
+  @attr('object') total;
```

Comment:
In my experience `@attr('object')` is less supported than just `@attr()` which leaves whatever object passed alone. We can leave this until implementation demands otherwise, but just an FYI of a potential gotcha

Reply:
This was `@attr('object')` in the last release, so I don't _think_ it caused any problems? But good FYI